git_creation_and_tracking/git_creation_and_tracking.md

```diff
@@ -0,0 +1,829 @@
+</div>
+
+If you are not sure if Git is configured on your computer, enter `git config user.name` into your command line interface and see if it returns your name. You can also check that it it has the correct email for you with `git config user.email`. If these are correct you are all ready to start the module!
+
+If you need to get Git configured on your computer there is a module to help you get set up.  Depending on what computer you are using, use the instructions for a [Mac or Linux operating system](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/setting_up_git_mac_and_linux/setting_up_git_mac_and_linux.md#1), or a [Windows operating system](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/git_setup_windows/git_setup_windows.md#1)
+
+## Creating a Repository
+
+Usually you will use Git to track the progress of a project. As we learn about Git, we will use the story of Wolfman and Dracula who are investigating if it
+is possible to send a planetary lander to Mars.
+
+In the examples, we will see code as if it is written by Vlad Dracula. If you are practicing these steps as you move through this module, your name and email address will appear instead of Dracula's.
+
+To copy the code in the examples, enter everything after the dollar sign "$" into the command line, and then press the enter or return key. If the command has any output, it will not start with a dollar sign, but a new line where you can type after a dollar sign will appear.
+
+```console
+$logname
+vlad
+$
+```
+
+The **code block** above tells you that when Dracula types `logname` and presses enter, the output is 'vlad', his username. Try entering `logname` in your own console. There will be other code in front of each `$` describing who and where you are. Since the stuff before the `$` will be different for everyone, it is customary to omit it from example code. We will also omit the `$` after the output.
+
+### Initializing the repository
+
+First, let's create a directory folder for our work. Depending on how your computer is organized the `Desktop` on Mac or Linux, or in `C` Drive on Windows machines may be good locations for the new directory folder, but you can put it wherever you like. Dracula's computer has a `Desktop` where he does his work.
+
+
+```console
+$ cd ~/Desktop
+$ mkdir planets
+$ cd planets
+```
+The `cd` command means **change directory**.  In our first command, we're changing directory to our home directory (that's the `~`, and within that, to the directory called `Desktop`.  If you don't have this directory in your file system, you'll get an error.  That's okay -- just navigate to a different directory you'd like to work in (as long as that directory isn't within a git repository).  The `mkdir` command tells the computer to **make** a **directory** with the name "planets." Once we have a `planets` directory, we use `cd` open that directory.
```

Comment:
Tiny copy edit comment! Add in a second parenthesis after "...repository)". add a "to" after "...directory, we use `cd`".

Reply:
added a `)` after the `~` which I think should have had one to begin with!